tpe/lib/src/Link.hh

```diff
@@ -42,6 +44,39 @@ class IGNITION_PHYSICS_TPELIB_VISIBLE Link : public Entity
   /// \brief Add a collision
   /// \return Newly created Collision
   public: Entity &AddCollision();
+
+  /// \brief Set the linear velocity of link
+  /// \param[in] _velocity linear velocity
+  public: void SetLinearVelocity(const math::Vector3d &_velocity);
```

Comment:
It would be nice to document the unit and frame of these velocities.

Reply:
updated doc in 58f3e0b and ab04c90